lib/index.es6

```diff
@@ -0,0 +1,97 @@
+import * as path from "path";
+import * as fs from "fs";
+
+import { watch } from "chokidar";
+import {sync as mkdirp} from "mkdirp";
+import most from "most";
+import _ from "lodash";
+
+import compile from "./compile";
+import { entries } from "./util";
+import loadAst from "./compile/modules/load-ast";
+import compileModules from "./compile/modules/compile";
+
+export default function Interlock (options) {
```

Comment:
Any reason not to use a class here? You're a) expecting people to instantiate it and b) attaching to the prototype. Seems a `class` would accomplish the exact same thing here with a slightly more concise syntax.